WordPress/Classes/ViewRelated/Comments/CommentHeaderTableViewCell.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+class CommentHeaderTableViewCell: UITableViewCell, Reusable {
+
+    // MARK: Initialization
+
+    required init(reuseIdentifier: String = CommentHeaderTableViewCell.defaultReuseID) {
```

Comment:
I don't see that `reuseIdentifier` needs to be a parameter. `Reusable` has its own `defaultReuseID`, as you're defaulting to. I don't see any reason to allow it to be overridden. Maybe it should just be passed directly to `super.init`?


```
required init() {
    super.init(style: .subtitle, reuseIdentifier: CommentHeaderTableViewCell.defaultReuseID)
    configureStyle()
}
```

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17095/commits/e2c33df7d059541b8aa688fe8e4b98839aa114d6 !